### PR TITLE
chore(updates.jenkins.io): use new `mirrorbits-parent` helm chart

### PR DESCRIPTION
**Pull request by lemeurherve**
This PR use [the new `mirrorbits-parent`](https://github.com/jenkins-infra/helm-charts/tree/main/charts/mirrorbits-parent) helm chart instead of [the `mirrorbits` one](https://github.com/jenkins-infra/helm-charts/tree/main/charts/mirrorbits) to deploy updates.jenkins.io

Content of the secret:
```yaml
mirrorbits-lite:
    geoipupdate:
        account_id: THE_ID
        license_key: THE_KEY
    repository:
        secrets:
            enabled: true
            data:
                azurestorageaccountkey: THE_KEY
                azurestorageaccountname: THE_NAME
    conf: |
        Repository: /srv/repo
        Templates: /usr/share/mirrorbits/templates
        GeoipDatabasePath: /usr/share/GeoIP/
        Gzip: true
        ListenAddress: :8080
        #RPCListenAddress: localhost:3390
        RedisAddress: updates-jenkins-io.redis.cache.windows.net:6379
        # [...snip...]
```

Ref: https://github.com/jenkins-infra/helpdesk/issues/2649

### CODE REVIEW

File: config/updates.jenkins.io-parent.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  resources:
+    limits:
+      cpu: 100m
+      memory: 128Mi
+    requests:
+      cpu: 50m
+      memory: 64Mi
+
+  nodeSelector:
+    agentpool: x86medium
```
Comment (dduportal) [inline config/updates.jenkins.io-parent.yaml:116]:
```suggestion
rsyncd:
  enabled: true

  configuration:
    components:
      - name: jenkins
        path: /rsyncd/data/jenkins
        comment: "Jenkins Read-Only Mirror"
        volume:
          persistentVolumeClaim:
            claimName: updates-jenkins-io-binary

  resources:
    limits:
      cpu: 100m
      memory: 128Mi
    requests:
      cpu: 100m
      memory: 128Mi

  podSecurityContext:
    runAsUser: 65534 # User 'nobody'
    runAsGroup: 65534 # Group 'nogroup'
    runAsNonRoot: true

  containerSecurityContext:
    readOnlyRootFilesystem: true
    allowPrivilegeEscalation: false
    capabilities:
      drop:
      - ALL

  nodeSelector:
    agentpool: x86medium

```

- Requires https://github.com/jenkins-infra/helm-charts/pull/689, https://github.com/jenkins-infra/helm-charts/pull/688 and https://github.com/jenkins-infra/helm-charts/pull/687 to be merged (and the release 1.2.0 deployed, and the lock file updated here)
- Set up same rexsource requests and limits
- Harden execution by setting security contexts

**Comment by lemeurherve** [comment]
Unrelated updatecli error 🤔 

image

**Comment by dduportal** [comment]
> Unrelated updatecli error 🤔
> image

If you look at the error associated with `checkPodTemplate` :

```
WARNING: current yaml key is "spec.containers[0].name" and should be updated to "$.spec.containers[0].name"
 ✗ key "$.spec.containers[0].name", in YAML file "PodTemplates.yaml", is incorrectly set to "\"jnlp\"" and should be "jnlp"
```

I believe if you solve the warning, then updatecli will behave (unrelated to your PR)

**Comment by lemeurherve** [comment]
Updatecli now fails only on `mirrorbits-parent` target (expected as not yet on main branch), merging.

**Pull request merged**